### DISCUSSION TITLE
使用react-query-kit报错

### DISCUSSION THREAD

**Discussion by pangxzi** [umijs/umi#12457] [Q&A]
![image](https://github.com/umijs/umi/assets/52907799/d87293d0-7596-4655-ac77-7e36c61ba8d9)
![image](https://github.com/umijs/umi/assets/52907799/1aa73fe9-b959-4e6f-a2a9-3dbd408e68b2)
![image](https://github.com/umijs/umi/assets/52907799/71eddafc-3f1c-47df-895b-c7fc125efa0f)

**是需要自己加`provider`吗？**

**Comment by fz6m** [comment]
现在 umi 4 内置这个 react query 插件，是默认使用的内置的 RQ v4 版本，而最新的 RQ 是 v5 版本，如果需要使用 v5 版本，需要自己安装 RQ v5 和 RQ devtools v5 版本到项目里（不使用 devtools 可以不安装）。

如还有问题，给一个最小复现看看吧。
